packages/wasm/src/core/realm.js

```diff
@@ -1,10 +1,21 @@
-import Realm from 'realm';
 import {Account, TokenPrice, RequestLog} from './realm-schemas';
 
+let Realm;
 let realm;
 
 const schema = [TokenPrice, Account, RequestLog];
 
+/**
+ *
+ * Realm is being used by the dock-app
```

Comment:
if Realm is only intended to be used by the dock app should we move the code? Or is the move to typeorm soon enough that it's not worth it?

Reply:
We should move to typeorm soon enough. That's not a big lift, so could be handled in the next month. https://dock-team.atlassian.net/browse/DCKA-1699